hazelcast/src/main/java/com/hazelcast/projection/impl/SingleAttributeProjection.java

```diff
@@ -35,24 +39,27 @@
  *
  * @param <I> type of the input
  */
+@SuppressFBWarnings("SE_NO_SERIALVERSIONID")
```

Comment:
Added suppression as this class is already IdentifiedDataSerializable. It did not have serialversionid before. We may add the serial version id now. Not sure if necessary in 4.0 though?